File: AkitaModelJava/akitamodel/src/main/java/skyware/lis/akitamodel/Department.java
package skyware.lis.akitamodel;

import lombok.Data;

/**
 * Department is a group of tests (analytes) and panels/profiles.
 */
@Data
public class Department {

    /**
     * Primary key.
     */
    private int id;

    /**
     * Name of the department.
     */
    private String name;

    /**
     * Label to be shown on results report for items in this department.
     */
    private String rangeText;

    /**
     * Sorting value.
     */
    private int rank;
    
}

File: AkitaMotelTS/src/Department.ts
/**
 * Skyware.Lis.AkitaModel
 */

/**
 * Department is a group of tests (analytes) and panels/profiles.
 */
export class Department {
  /**
   * Primary key.
   */
  id!: number;

  /**
   * Name of the department.
   */
  name?: string;

  /**
   * Label to be shown on results report for items in this department.
   */
  rangeText?: string;

  /**
   * Sorting value.
   */
  rank!: number;
}
